src/lib/metaphysics.coffee

```diff
@@ -2,13 +2,20 @@ Q = require 'bluebird-q'
 qs = require 'qs'
 request = require 'superagent'
 { extend, some } = require 'underscore'
-{ METAPHYSICS_ENDPOINT, API_REQUEST_TIMEOUT, REQUEST_ID } = require('sharify').data
+{ METAPHYSICS_ENDPOINT, METAPHYSICS_BLUE_LOAD_RATIO, METAPHYSICS_BLUE_ENDPOINT, API_REQUEST_TIMEOUT, REQUEST_ID } = require('sharify').data
+
+oneIn = (n) ->
+  Math.floor(Math.random() * n) is 0
+
+service = ->
+  return METAPHYSICS_ENDPOINT unless METAPHYSICS_BLUE_ENDPOINT? and METAPHYSICS_BLUE_LOAD_RATIO? > 0
```

Comment:
Not super up on my coffeescript but why is the right-hand side of this `and` not `METAPHYSICS_BLUE_LOAD_RATIO > 0` (without the `?`) as `METAPHYSICS_BLUE_LOAD_RATIO` is defined as an integer.

Reply:
👍 no that's fine to write it safely was just wondering!